crud/common/utils.lua

```diff
@@ -140,6 +140,15 @@ function utils.unflatten(tuple, space_format)
     return object
 end
 
+function utils.extract_sharding_key(tuple, fieldnos)
+    local key = {}
+    for _, fieldno in ipairs(fieldnos) do
+        key[fieldno] = tuple[fieldno]
```

Comment:
I guess it should be `key[i] = tuple[fieldno]`

Reply:
function `utils.extract_sharding_key()` has gone